source/jormungandr/jormungandr/interfaces/v1/LineReports.py

```diff
@@ -84,6 +88,11 @@ def __init__(self):
                                 schema_metadata={'format': 'pt-object'})
         parser_get.add_argument("disable_geojson", type=BooleanType(), default=False,
                                 help="remove geojson from the response")
+        parser_get.add_argument("since", type=DateTimeFormat(),
+                                help="filters disruptions valid after this date")
```

Comment:
I'd replace "filters" by "use" (same below).
With filters, we don't know if we remove, or keep (we keep).